src/Filter/AbstractDateFilter.php

```diff
@@ -73,7 +73,31 @@ public function filter(ProxyQueryInterface $queryBuilder, $alias, $field, $data)
             $endDateParameterName = $this->getNewParameterName($queryBuilder);
 
             if (DateRangeType::TYPE_NOT_BETWEEN == $data['type']) {
-                $this->applyWhere($queryBuilder, sprintf('%s.%s < :%s OR %s.%s > :%s', $alias, $field, $startDateParameterName, $alias, $field, $endDateParameterName));
+                if ($data['value']['start'] and $data['value']['end']) {
```

Comment:
Please use `&&`

Reply:
can this be ensured by the php-cs-fixer? 🤔